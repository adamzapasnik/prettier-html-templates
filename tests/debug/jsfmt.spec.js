run_spec(__dirname, ['test']);
