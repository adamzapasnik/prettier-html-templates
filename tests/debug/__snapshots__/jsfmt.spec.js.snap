// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`s.html.test 1`] = `
====================================options=====================================
parsers: ["test"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<label>Post to Changelog News <%= help_icon("Disable to publish to audio feed only.") %></label>

=====================================output=====================================
<label
  >Post to Changelog News
  <%= help_icon("Disable to publish to audio feed only.") %>
</label>

================================================================================
`;
